nautobot/utilities/forms/fields.py

```diff
@@ -7,18 +7,18 @@
 from django import forms
 from django.apps import apps
 from django.contrib.contenttypes.models import ContentType
-from django.forms.fields import JSONField as _JSONField, InvalidJSONInput
+from django.contrib.postgres.forms import SimpleArrayField
```

Comment:
Presumably we're wanting/needing to move away from a Postgres-specific field type in the near future - any thoughts here?

Reply:
I thought about it, but this form field isn't specific to Postgres, so for now I think it's okay to leave using this field. The source code of that field is actually doing a lot, so I didn't think it was worth replacing right now.